.github/workflows/main.yml

```diff
@@ -18,14 +18,56 @@ jobs:
           - macos-latest
           - ubuntu-latest
           # - windows-latest
-
     runs-on: ${{ matrix.os }}
-
     steps:
     - uses: actions/checkout@v2
+    - uses: actions/cache@v2
+      with:
+        path: |
+          ~/.cargo/registry
+          ~/.cargo/git
+          target
+        key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
     - name: Build
       run: cargo build --verbose
+    - name: Build Release
+      run: cargo build --release --all-features --verbose
```

Comment:
Why not build `release` and `debug` in one command?

Reply:
Not sure how you'd do that. Also, the build processes are completely independent.

Reply:
Ah, my bad. I somehow thought `--release` was a target, but it's actually a compilation option.